docs/index.rst

```diff
@@ -56,6 +56,7 @@ against malware and other security risks. It is built on Qubes OS and requires a
    admin/reference/troubleshooting_updates
    admin/reference/provisioning_usb
    admin/reference/backup
+   admin/reference/thinkpad_bios
```

Comment:
For the sake of consistency, we should probably either have a small guide for Framework BIOS updates as well, or at least link out to their knowledgebase article on the topic: https://knowledgebase.frame.work/framework-laptop-bios-releases-S1dMQt6F

Reply:
Er, that link was for the 11th gen Intel board specifically. Looks like each product has its own BIOS update page. We'd either need to point to https://knowledgebase.frame.work/categories/products_en-r1aUmxtrd, or consider linking to each individual page in the aforementioned small guide.

Reply:
My pref would be to link wherever possible - if that's an option I'd link from the section in the hardware reccos. Where there's a long or involved process that we have to document locally, I'd prefer to have it in a separate page.